app.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const app = express();
+const { getAllTopics } = require('./controllers/topic.controller');
+app.use(express.json());
```

Comment:
This line of code should not be included here at this point as it doesn't serve any purpose for now.